runners/spark/src/main/java/org/apache/beam/runners/spark/metrics/MetricsAccumulator.java

```diff
@@ -37,7 +37,7 @@
  * For resilience, {@link AccumulatorV2 Accumulators} are required to be wrapped in a Singleton.
  *
  * @see <a
- *     href="https://spark.apache.org/docs/latest/api/java/org/apache/spark/util/AccumulatorV2.html">accumulatorsV2</a>
+ *     href="https://spark.apache.org/docs/2.4.4/streaming-programming-guide.html#accumulators-broadcast-variables-and-checkpoints">accumulatorsV2</a>
```

Comment:
Why the specific version of Spark is used here and not just "latest"?

Reply:
to be in sync with the version of spark we use otherwise if a new spark version comes out and we don't upgrade, then the javadoc is out of sync

Reply:
Ok, so we have not to forget to update these links once Spark deps version will be changed.